assured-workloads-example/storage.tf

```diff
@@ -0,0 +1,46 @@
+/**
+ * Copyright 2024 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+data "google_storage_project_service_account" "gcs_account" {
+  project = module.aw_workload_project.project_id
+}
+
+resource "google_kms_crypto_key_iam_binding" "gcs_service_account_binding" {
```

Comment:
Why not `google_kms_crypto_key_iam_member` instead?
Binding will override any existing permissions for this role - not sure if that's what we want